catchpoint-mixin/panels.libsonnet

```diff
@@ -0,0 +1,418 @@
+          description='Time taken to load and render content on the webpage.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      clientProcessing:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Client processing',
+          targets=[t.clientProcessing],
+          description='Client processing time, which reflects the time spent on client-side processing, including script execution and rendering.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      additionalDelay:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Additional delays',
+          targets=[t.additionalDelay, t.waitTime],
+          description='Additional delays encountered due to redirects, as well as time from successful connection to receiving the first byte.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      responseContentSize:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Response content size',
+          targets=[t.responseContentSize, t.responseHeaderSize],
+          description='Size of the HTTP response content.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('decbytes')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      totalContentSize:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Total content size',
+          targets=[t.totalContentSize, t.totalHeaderSize],
+          description='Total size of the HTTP response content and headers.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('decbytes')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      networkConnections:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Network connections',
+          targets=[t.networkConnections],
+          description='Number of connections made.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('conn')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      hostsContacted:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Hosts contacted',
+          targets=[t.hostsContacted],
+          description='Number of hosts contacted.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('hosts')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      cacheAccess:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Cache access',
+          targets=[t.cacheAccess],
+          description='Number of cached elements accessed.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
```

Comment:
I think this might benefit from including values in a table legend?

Also not sure if this panel was tested with the sample load, because there's no real data here 
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/3f06ec7b-0ee1-40a8-b2f7-e35eee7f69f7)

Reply:
Im not sure about the values, I could go for it if you think it would look nice. I am going to go ahead and setup the load gen for this metric real quick and i guess we can make a decision then.

Reply:
Having seen it with load gen, it's definitely better, but I still worry a little about it when it's scaled up with multiple labels.
Less so on this variant 
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/9f629e43-72f5-4caf-a490-a85c54db5286)


And more so on the "by test" variant where it's labelling the sites 
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/07878e50-e96b-44d6-9e22-17ad866cf322)


Honestly I think this is the sort of thing where we just have to go with one and adjust as we get feedback. I think I'm ok with it as is, but @mshahzeb what do you think?

Reply:
Yes I agree lets go with this one as is for now.